tests/ci/setup_slurm_and_run_tests.sh

```diff
@@ -90,6 +90,21 @@ fi
 
 export SIMPLE_CI=1
 
+sudo --user="${USER}" --login bash -c 'cd ${PWD}/tests; find ./ -name "*.log" -delete'
+
 # Always running at least with '--enable-modules'. No need to check for
 # an empty TESTS array.
 sudo --user="${USER}" --preserve-env=SIMPLE_CI --login bash -c "cd ${PWD}/tests; ./bootstrap; ./configure --disable-all --enable-modules --enable-rms-harness --with-mpi-families='openmpi4 mpich' ${TESTS[*]}; make check"
+
+tests_status=$?
```

Comment:
Not sure that this will work. The script is running with `set -e` which will result in an exit already here if the return code is not zero. So maybe a `set +e` is necessary before running the test?